gapil/analysis/reference_value.go

```diff
@@ -171,8 +171,10 @@ func (v *ReferenceValue) Clone() Value {
 func (v *ReferenceValue) field(s *scope, name string) Value {
 	candidates := make([]Value, 0, len(v.Assignments))
 	for a := range v.Assignments {
-		field := s.getInstance(a).(fieldHolder).field(s, name)
-		candidates = append(candidates, field)
+		if fh, ok := s.getInstance(a).(fieldHolder); ok {
```

Comment:
This is changed because of build failure with the original code.

The build fails with the original code if we drop the line 721 to line 745 in gapis/api/vulkan/api/descriptor.api, without any other changes. This is weird. 

The build does not happen before at commit: 7915ae323ed9dc1763b91088256ec9f996cd7728, which is the last commit that drops the line 721 to line 745 in gapis/api/vulkan/api/descriptor.api

Reply:
I'm going to try and reproduce this error - I don't think there should be a type panic here...

Reply:
Okay, after much beard scratching, I think I know what's going on here. See my comment below.